xds/data/orca/v3/orca_load_report.proto

```diff
@@ -21,9 +21,8 @@ message OrcaLoadReport {
   // resources. This should be derived from the latest sample or measurement.
   double mem_utilization = 2 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
 
-  // Total RPS being served by an endpoint. This should cover all services that an endpoint is
-  // responsible for.
-  uint64 rps = 3;
+  // Deprecated -- use qps instead.
```

Comment:
nit: keep old comment, and add the deprecation comment.

Reply:
Done.

Reply:
nit (for the rendered docs):
```suggestion
  // Deprecated -- use ``qps`` instead.
```